Explorer/Assets/Scripts/ECS/StreamableLoading/Common/Systems/LoadSystemBase.cs

```diff
@@ -241,7 +254,9 @@ private void ReportException(Exception exception)
                 // Set result for the reusable source
                 // Remove from the ongoing requests immediately because finally will be called later than
                 // continuation of cachedSource.Task.SuppressCancellationThrow();
+                int a = cache.OngoingRequests.Count;
```

Comment:
What is the int a?

Reply:
a forgotten thing I used to log xD removed it now :) thank you for spotting it!